tests/integration/mixer/outboundtrafficpolicy/helper.go

```diff
@@ -51,14 +57,11 @@ spec:
   - istio.io
   location: MESH_EXTERNAL
   ports:
-  - name: http
-    number: 80
-    protocol: HTTP
   - name: http-for-https
-    number: 443
+    number: 9443
```

Comment:
Please update comments on lines 238:267 which explains traffic flows based on the new semantics. Makes this this a bit more readable.

Reply:
I missed that comment since I started working on this before Jacobs PR landed, thanks! Iupdated to include the new traffic flows